packages/components/src/Header.tsx

```diff
@@ -178,12 +148,20 @@ export const Header = ({
                                 >
                                     twitter ↗
                                 </Button>
+                                {accountId && (
+                                    <Button
+                                        onClick={onSignOut}
+                                        variant="mobileMenu"
+                                    >
+                                        disconnect
+                                    </Button>
+                                )}
                             </Box>
                         )}
                     </Box>
                 </Box>
                 <a
-                    href={`/${base}/bids`}
+                    href="/"
```

Comment:
Let's have this not link to anything? Otherwise, it's confusing if on desktop it goes to bids and on mobile it goes to home.